src/python/pants/notes/1.27.x.rst

```diff
@@ -0,0 +1,852 @@
+1.27.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.27.x`` ``stable`` series.
+
+The ``1.27.x`` series deprecates several unused backends and plugins. Please
+see https://groups.google.com/forum/#!topic/pants-devel/ngMZRIJvwHM for more
+information.
+
+1.27.0rc0 (4/17/2020)
+----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Deprecate the v1 lambda task. (#9566)
+  `PR #9566 <https://github.com/pantsbuild/pants/pull/9566>`_
+
+* Remove several unused TargetAdaptor utility rules (#9569)
+  `PR #9569 <https://github.com/pantsbuild/pants/pull/9569>`_
+
+* Remove the fork_context method and support code. (#9554)
+  `PR #9554 <https://github.com/pantsbuild/pants/pull/9554>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Allow `target_types2` register.py entry point to register new symbols (#9558)
+  `PR #9558 <https://github.com/pantsbuild/pants/pull/9558>`_
+
+* Add link back to source control dir from physical workdir (#9531)
+  `PR #9531 <https://github.com/pantsbuild/pants/pull/9531>`_
+
+* Unrevert black change and fix issue with it (#9545)
+  `PR #9545 <https://github.com/pantsbuild/pants/pull/9545>`_
+
+* Revert "quiet black formatter in CI (#9536)" (#9542)
+  `PR #9536 <https://github.com/pantsbuild/pants/pull/9536>`_
+  `PR #9542 <https://github.com/pantsbuild/pants/pull/9542>`_
+
+* quiet black formatter in CI (#9536)
+  `PR #9536 <https://github.com/pantsbuild/pants/pull/9536>`_
```

Comment:
Internal tooling. These don't impact end users. You could put them under testing too.